cmd/stop.go

```diff
@@ -48,6 +82,15 @@ dapr stop --app-id <ID>
 
 func init() {
 	StopCmd.Flags().StringVarP(&stopAppID, "app-id", "a", "", "The application id to be stopped")
+	StopCmd.Flags().StringVarP(&runFilePath, "run-file", "f", "", "Path to the run template file for the list of apps to stop")
 	StopCmd.Flags().BoolP("help", "h", false, "Print this help message")
 	RootCmd.AddCommand(StopCmd)
 }
+
+func executeStopWithRunFile(runFilePath string) error {
+	absFilePath, err := filepath.Abs(runFilePath)
+	if err != nil {
+		return fmt.Errorf("failed to get absolute file path for %s: %w", runFilePath, err)
+	}
+	return standalone.StopAppsWithRunFile(absFilePath)
```

Comment:
can pass in the list of apps from the run function above. 
Query List() only once?

Reply:
it is now querying only once.
for `dapr stop -f ` -> see method `StopAppsWithRunFile`
for normal `dapr stop appID` -> see line no 66 in cmd/stop.go.